examples/oauth/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "bolt-oauth-example",
+  "version": "1.0.0",
+  "description": "Example app using OAuth",
+  "main": "app.js",
+  "scripts": {
+    "start": "node app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Slack Technologies, Inc.",
+  "license": "MIT",
+  "dependencies": {}
```

Comment:
Do we need bolt-js as a dependency?

Reply:
As this example requires the latest revision, I used `npm link` for adding the dependency for now. But, to avoid confusion, I just added it to the dependencies.